Lib/test/test_itertools.py

```diff
@@ -590,6 +590,26 @@ def test_count_with_stride(self):
         self.assertEqual(type(next(c)), int)
         self.assertEqual(type(next(c)), float)
 
+    def test_count_threading(self, step=1):
```

Comment:
I think the WASI build needs something like:

```suggestion
    @threading_helper.requires_working_threading()
    def test_count_threading(self, step=1):
```